underlay/src/main/java/bio/terra/tanagra/underlay/serialization/SZUnderlay.java

```diff
@@ -118,6 +118,23 @@ public class SZUnderlay {
               + "criteria definitions for standing up a new underlay.")
   public Set<String> prepackagedDataFeatures;
 
+  @AnnotatedField(
+      name = "SZUnderlay.visualizations",
+      markdown =
+          "List of paths of all the visualizations.\n\n"
+              + "A visualization contains all of the configuration to display a population level visualization "
```

Comment:
Does "population level" mean "cohort level"? I think the two levels we have are cohort and underlay/dataset.